plugins/secretstores/systemd/README.md

```diff
@@ -0,0 +1,248 @@
+via `LoadCredentialEncrypted` in the service file. This is the case for the
+Telegraf service provided in this repository. It expects encrypted credentials
+to be stored in `/etc/telegraf/credentials`.
+
+It is important to note that credentials can only be created and used on
+the **same machine** as decrypting the secrets requires the encryption
+key *and* a key stored in TPM2. Therefore, creating credentials and then
+copying it to another machine will fail!
+
+Please be aware that, due to its nature, this plugin is **ONLY** available
+when started as a service. It does **NOT** find any credentials when started
+manually via the command line! Therefore, `secrets` commands should **not**
+be used with this plugin.
+
+## Usage <!-- @/docs/includes/secret_usage.md -->
+
+Secrets defined by a store are referenced with `@{<store-id>:<secret_key>}`
+the Telegraf configuration. Only certain Telegraf plugins and options of
+support secret stores. To see which plugins and options support
+secrets, see their respective documentation (e.g.
+`plugins/outputs/influxdb/README.md`). If the plugin's README has the
+`Secret-store support` section, it will detail which options support secret
+store usage.
+
+## Configuration
+
+```toml @sample.conf
+# Secret-store to access systemd secrets
+[[secretstores.systemd]]
+  ## Unique identifier for the secretstore.
+  ## This id can later be used in plugins to reference the secrets
+  ## in this secret-store via @{<id>:<secret_key>} (mandatory)
+  id = "my_secretstore"
+
+  ## Path to systemd credentials directory
+  ## This should not be required as systemd indicates this directory
+  ## via the CREDENTIALS_DIRECTORY environment variable.
+  # path = "${CREDENTIALS_DIRECTORY}"
+```
+
+Each Secret provided by systemd will be available as file under
+`${CREDENTIALS_DIRECTORY}/<secret-name>` for the service. You will **not**
+be able to see them as a regular user, not are those files accessible to other
+services.
+
+## Credential management
+
+Most steps here are condensed from the [systemd-creds man-page][systemd-creds]
+and are using this command. Please also check that man-page as the options
+or verbs used here might be outdated for the systemd version you are using.
+
+**Please note**: We are using `/etc/telegraf/credentials` as our storage
+location for encrypted credentials throughout the examples below. This is
+because the Telegraf service file expects credentials to be located there
+by default. Furthermore, we assume the secret-store ID to be set to `syscreds`
+in the examples.
+
+### Setup
+
+If you have not done it already, systemd requires a first-time setup of the
+credential system. If you are planning to use the TPM2 chip of your system
+for protecting the credentials, you should first make sure that it is
+available using
+
+```shell
+# sudo systemd-creds has-tpm2
+partial
+-firmware
++driver
++system
++subsystem
+```
+
+The output should look similar to the above.
```

Comment:
I suggest adding:
"The output should look similar to the above. If it says "yes" to TPM support being available then credentials can also be tied to the device by utilizing TPM sealing. See the `systemd-creds` man page for details."